app/frontend/src/service/events.ts

```diff
@@ -86,3 +90,55 @@ export async function setEventAttendance({
   const res = await client.events.setEventAttendance(req);
   return res.toObject();
 }
+
+interface EventInput {
+  content: string;
+  photoKey?: string;
+  title: string;
+  startTime: Date;
+  endTime: Date;
+}
+
+interface OnlineEventInput extends EventInput {
+  isOnline: true;
+  parentCommunityId: number;
+  link: string;
+}
+
+interface OfflineEventInput extends EventInput {
+  isOnline: false;
+  address: string;
+  lat: number;
+  lng: number;
+}
+
+export type CreateEventInput = OnlineEventInput | OfflineEventInput;
+
+export async function createEvent(input: CreateEventInput) {
+  const req = new CreateEventReq();
+  req.setTitle(input.title);
+  req.setContent(input.content);
+  req.setStartTime(Timestamp.fromDate(input.startTime));
+  req.setEndTime(Timestamp.fromDate(input.endTime));
+  req.setTimezone("UTC");
```

Comment:
Why UTC? I think this should be empty/not set: `// the tzdata timezone identifier, empty string means tz of creating user`

Reply:
I'm a little confused about this tbh, since `Timestamp.fromDate` always converts thing back to UTC so that's why I set it... but maybe it's fine to leave it out? 🤷  The frontend also doesn't use it at the moment, it converts the `startTime`/`endTime` from `Timestamp` back to a `Dayjs` object and figures out the correct local time to display there (currently with the user's browser timezone, but could be updated to use the user profile's timezone).

So this separate timezone field feels a bit redundant?